src/admin/config.yml

```diff
@@ -1,24 +1,23 @@
-# local_backend:
-#   url: http://localhost:8082/api/v1
-
-# backend:
-#   name: git-gateway
+local_backend:
+  url: http://localhost:8082/api/v1
 
 backend:
-  name: github
-  repo: kindsys/digital-services-coalition
-  branch: main
+  name: git-gateway
+
+# backend:
+#   name: github
+#   repo: kindsys/digital-services-coalition
+#   branch: main
```

Comment:
When creating a PR, let's comment out the local dev stuff and keep the production config